platform-configs/include/protectli-common.robot

```diff
@@ -51,7 +50,7 @@ ${CUSTOM_NETWORK_BOOT_ENTRIES_SUPPORT}=         ${TRUE}
 ${EXTERNAL_HDMI_DISPLAY_SUPPORT}=               ${TRUE}
 ${EXTERNAL_DISPLAY_PORT_SUPPORT}=               ${TRUE}
 ${CUSTOM_LOGO_SUPPORT}=                         ${TRUE}
-${UEFI_SHELL_SUPPORT}=                          ${TRUE}
+${UEFI_SHELL_SUPPORT}=                          ${FALSE}
```

Comment:
We want to drop UEFI shell from all dashro releases moving forward.

So we can even move it to default.robot